Base/usr/share/man/man1/Applications/ImageViewer.md

```diff
@@ -20,13 +20,23 @@ File manipulation has no effect on the image. Flip or rotate actions are not sav
 
 ImageViewer is even smart enough to detect other images and display them when clicking on the navigation buttons or when using direction arrows. ImageViewer can even set the currently loaded image as a Desktop Wallpaper.
 
+## Options
+
+* `-f`, `--fallback-binary`: Try to decode a file as an image of a known format, otherwise fallback to use it as a raw binary bitmap data
+* `-w Width`, `--width Width`: Force width when displaying an image
+* `-h Height`, `--height Height`: Force height when displaying an image
```

Comment:
Width/Height -> width/height

I.e. similar to `bitmap_format` and matches the arguments as provided to `ArgsParser`